Windows: runDistributable fails with file not found

Hi,

the `runDistributable` gradle task fails on windows with the following exception:
```
* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':ui:runDistributable'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:208)
<...>
Caused by: org.gradle.process.internal.ExecException: A problem occurred starting process 'command 'ui.exe''
<...>
Caused by: net.rubygrapefruit.platform.NativeException: Could not start 'ui.exe'
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:27)
        at net.rubygrapefruit.platform.internal.WindowsProcessLauncher.start(WindowsProcessLauncher.java:22)
        at net.rubygrapefruit.platform.internal.WrapperProcessLauncher.start(WrapperProcessLauncher.java:36)
        at org.gradle.process.internal.ExecHandleRunner.startProcess(ExecHandleRunner.java:98)
        at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:71)
        ... 4 more
Caused by: java.io.IOException: Cannot run program "ui.exe" (in directory "C:\Repositories\t2-interceptor\ui\build\compose\binaries\main\app\ui"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:25)
        ... 8 more
Caused by: java.io.IOException: CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        ... 9 more
```

Looking at the mentioned directory, I can see the file being present.
```
dir C:\Repositories\t2-interceptor\ui\build\compose\binaries\main\app\ui

12.02.2021  17:38    <DIR>          .
12.02.2021  17:38    <DIR>          ..
12.02.2021  17:38    <DIR>          app
12.02.2021  17:38    <DIR>          runtime
12.02.2021  17:38           434.176 ui.exe
12.02.2021  17:38            25.214 ui.ico
               2 Datei(en),        459.390 Bytes
               4 Verzeichnis(se), 39.665.311.744 Bytes frei
```
I´m using `0.3.0-build152`

Best regards,
Fabian

--- a/gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/tasks/AbstractRunDistributableTask.kt
+++ b/gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/tasks/AbstractRunDistributableTask.kt
@@ -43,7 +43,7 @@ abstract class AbstractRunDistributableTask @Inject constructor(
 
         execOperations.exec { spec ->
             spec.workingDir(workingDir)
-            spec.executable(executable)
+            spec.executable(workingDir.resolve(executable).absolutePath)
         }.assertNormalExitValue()
     }
 }

